packages/server/src/integrations/googlesheets.ts

```diff
@@ -286,7 +286,7 @@ class GoogleSheetsIntegration implements DatasourcePlus {
   async createTable(name?: string) {
     try {
       await this.connect()
-      return await this.client.addSheet({ title: name })
+      return await this.client.addSheet({ title: name, headerValues: ["test"] })
```

Comment:
Is this a real value used by google, or something added for testing in dev? would an empty array produce the same fix?

Reply:
This is just part of the `google-spreadsheet` library - we have to put a header in the new sheet, or we are unable to fetch the schema for the new table (sheet) and it breaks the builder